react/features/feedback/components/styles.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { BoxModel, ColorPalette, createStyleSheet } from '../../base/styles';
+
+export default createStyleSheet({
+    dialogComponent: {
+        paddingHorizontal: BoxModel.padding * 2,
+        paddingVertical: BoxModel.padding
+    },
+
+    feedbackHeader: {
+        minWidth: 300,
```

Comment:
I've seen this number before in this PR. It needs a constant I think. But more importantly, what about tablets? let's talk about it.

Reply:
I agree about the constants, did it. But also added a note to the constant to explain why it makes sense